java-client/pom.xml

```diff
@@ -470,6 +470,36 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.thrift</groupId>
+        <artifactId>thrift-maven-plugin</artifactId>
+        <version>0.10.0</version>
```

Comment:
For compatibility reason, it's recommend to use 0.11.0 version.

Reply:
This is plugin version, i found this is the latest version. thrift java lib version will be 0.11.0 which defined in libthrift.version